src/Select.js

```diff
@@ -1,9 +1,9 @@
 // @flow
 
 import React, { Component, type ElementRef, type Node } from 'react';
-
 import memoizeOne from 'memoize-one';
-import createEmotion, { type Emotion } from 'create-emotion';
```

Comment:
removed createEmotion in favor of emotion 10s CacheProvider and createCache for adding nonce support.